QuickDeployment/V2/QuickDeploymentEnvironment.md

```diff
@@ -2,100 +2,112 @@
 
 ## Summary
 
-This is an overview on the Chocolatey Quick Deployment Environment (QDE). It provides a single virtual machine appliance to be imported into your hypervisor-of-choice, which contains most of the various components of a Chocolatey organizational solution.
+This is an overview on the Chocolatey Quick Deployment Environment (QDE).
+It provides a single virtual machine appliance to be imported into your hypervisor-of-choice, which contains most of the various components of a Chocolatey organizational solution.
 
-![QDE Architechture](images/quickdeploy/QDE-architecture.gif)
+> :warning: **Warning**
+>
+> This solution targets environments up to about **1000 nodes**.
+> If you need a solution for a larger environment, QDE is generally suitable only as a proof-of-concept.
+> For best results, we would recommend a distributed infrastructure, separating each component into its own discrete node.
+> If you find yourself in need of a more scalable solution, please contact Support and we'll be more than happy to provide guidance for larger solutions.
+
+![QDE Architecture](images/quickdeploy/QDE-architecture.gif)
 
 <!-- TOC depthFrom:2 -->
 
 - [Summary](#summary)
 - [QDE Components](#qde-components)
 - [Getting QDE](#getting-qde)
-- [Links](#links)
 - [Related Articles](#related-articles)
 - [FAQ](#faq)
-  - [What does the QDE (Quick Deployment Environment) include?](#what-does-the-qde-quick-deployment-environment-include)
-  - [How much time does this save?](#how-much-time-does-this-save)
   - [How do we take advantage of QDE?](#how-do-we-take-advantage-of-qde)
-  - [We're already a C4B customer?](#were-already-a-c4b-customer)
+  - [How much time does this save?](#how-much-time-does-this-save)
+  - [What if we're already a C4B customer?](#what-if-were-already-a-c4b-customer)
   - [How do we upgrade QDE?](#how-do-we-upgrade-qde)
   - [What if we have a larger environment? (> 1k nodes)](#what-if-we-have-a-larger-environment--1k-nodes)
   - [Can we brag about how fast we were able to get configured?](#can-we-brag-about-how-fast-we-were-able-to-get-configured)
+- [See Also](#see-also)
```

Comment:
Renaming this is okay, but also make sure the anchor https://chocolatey.org/docs/quick-deployment-environment#links continues to work to get here as well. Also I was under the impression this was to move up before FAQ. That may not have happened yet and might still be on your list.

Reply:
Thought I did... ah, TOC mightn't've updated.

Hmm, fair point about the existing anchor. Don't think there's a (good) way to have that work in Markdown without adding a spot of HTML, so might be better to leave it the original name?

Reply:
For simplicity lets just leave it as the original name `links`?

Reply:
Yeah that makes the most sense atm. This is already done, then. 🙂

Reply:
This is resolved, then; left  `Links` as the header there.